.github/renovate.json5

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base", // Everything that renovatebot supports by default
+    // Renovate create an issue where all the renovate PRs are listed.
+    // The rule below should disable that issue.
+    ":disableDependencyDashboard"
+  ],
+  "regexManagers": [
+    {
+      // Update minikube action parameters
+      "fileMatch": [ "^.github/workflows/lint-charts.yaml$" ],
```

Comment:
This seems to point to the file that this PR deleted

Reply:
You are right. Solved (allegedly) in [#290b782](https://github.com/txqueuelen/pdns-stateless/pull/8/commits/290b782dd667a6bfcf8b6de6783eb7403ecdfed2) if I do not force-push again.